habitat-lab/habitat/tasks/rearrange/multi_task/composite_sensors.py

```diff
@@ -177,46 +178,6 @@ def update_metric(self, *args, episode, task, observations, **kwargs):
         self._prev_holding_obj = is_holding_obj
 
 
-@registry.register_measure
-class DoesWantTerminate(Measure):
-    cls_uuid: str = "does_want_terminate"
-
-    @staticmethod
-    def _get_uuid(*args, **kwargs):
-        return DoesWantTerminate.cls_uuid
-
-    def reset_metric(self, *args, **kwargs):
-        self.update_metric(*args, **kwargs)
-
-    def update_metric(self, *args, task, **kwargs):
-        self._metric = task.actions["rearrange_stop"].does_want_terminate
-
-
-@registry.register_measure
-class CompositeBadCalledTerminate(Measure):
```

Comment:
Why was this one removed, it looks like it is still used in set_able and rearrange tasks

Reply:
I just moved it to `rearrange_sensors.py`.